### PR TITLE
Update installEdgeDriver.ps1 to fetch Edge version dynamically

### PR DESCRIPTION
**Pull request by gladjohn**
**Fixes #**
OneBranch build failure

**Changes proposed in this request :** 
gets the edge version of the build machine and uses the version to download the edge driver.

**Testing :** 
builds and local powershell

**Performance impact :** 
none

**Documentation :** 
n/a

### CODE REVIEW

File: build/installEdgeDriver.ps1 [hunk 1]
```diff
@@ -1,4 +1,7 @@
-$url = "https://msedgedriver.azureedge.net/119.0.2151.72/edgedriver_win64.zip" #Chrome Driver from https://developer.microsoft.com/en-us/microsoft-edge/tools/webdriver/
+# Get the installed version of Microsoft Edge
+$edgeVersion = (Get-Item "C:\Program Files (x86)\Microsoft\Edge\Application\msedge.exe").VersionInfo.ProductVersion
```
Comment (bgavrilMS) [inline build/installEdgeDriver.ps1:2]: Can you error out with some message if this is null?

Comment (bgavrilMS) [inline build/installEdgeDriver.ps1:2]: Use %PROGRAMFILES(x86)% to avoid hardcoding ?

File: build/installEdgeDriver.ps1 [hunk 2]
```diff
@@ -1,4 +1,7 @@
...
+
+$url = "https://msedgedriver.azureedge.net/$edgeVersion/edgedriver_win64.zip" #Chrome Driver from https://developer.microsoft.com/en-us/microsoft-edge/tools/webdriver/
```
Comment (bgavrilMS) [inline build/installEdgeDriver.ps1:4]:
```suggestion
$url = "https://msedgedriver.azureedge.net/$edgeVersion/edgedriver_win64.zip" #Edge Driver from https://developer.microsoft.com/en-us/microsoft-edge/tools/webdriver/
```

File: build/installEdgeDriver.ps1 (hunk 1 above)
Comment (gladjohn) [reply build/installEdgeDriver.ps1]: added error checks

**Review by pmaytak — APPROVED** [review]

**Pull request merged**